### PR TITLE
Fix csproj

### PR DESCRIPTION
**Pull request by KenitoInc**
### Issues

When I rebased my fork, I got this diff.
Probably caused by this commit https://github.com/OData/odata.net/commit/bae11a51b2e0be642e2e42351774f14508becf55

### Description

*Briefly describe the changes of this pull request.*

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: src/Microsoft.OData.Core/Microsoft.OData.Core.csproj [hunk 1]
```diff
@@ -77,20 +77,6 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>
```
Comment (xuzhg) [inline src/Microsoft.OData.Core/Microsoft.OData.Core.csproj:80]: why do we remove them? They are needed for the resource generation

Comment (KenitoInc) [reply src/Microsoft.OData.Core/Microsoft.OData.Core.csproj:80]: Ping @corranrogue9

Comment (corranrogue9) [reply src/Microsoft.OData.Core/Microsoft.OData.Core.csproj:80]: I recall having an intermediate commit in my PR where I had accidentally removed this section, but these changes are not in the master branch, and I agree with Sam that they should stay.

Comment (KenitoInc) [reply src/Microsoft.OData.Core/Microsoft.OData.Core.csproj:80]: @corranrogue9  This [commit](https://github.com/OData/odata.net/commit/bae11a51b2e0be642e2e42351774f14508becf55) is in the master branch and it did some changes to csproj. Kindly have a look

**Pull request closed without merging**